dataland-inbound-proxy/config/utils/qaErrors.conf

```diff
@@ -4,5 +4,6 @@ location = /qa/json_errors.json {
   ssi on;
   ssi_types application/json;
   internal;
-  root /var/www/html;
+  root /var/www/html/error_documents;
+  try_files /json_errors.json =500;
```

Comment:
Was macht das genau?

Reply:
Die error pages haben nicht funktioniert, weil die versuchen auf /api/json_errors.json in dem Verzeichniss zuzugreifen. Mit try files leite ich das auf /json_errors.json um. Damit klappt das dann wieder

Reply:
Und was macht der =500 Teil?

Reply:
Wenn die Datei `json_errors.json` aus irgendwelchen Gründen nicht geunden wird, wird ein 500 - Internal Server Errror zurückgegeben (siehe http://nginx.org/en/docs/http/ngx_http_core_module.html#try_files). Es ist zwingend notwendig mindestens zwei Möglichkeiten angeben, deshalb habe ich das `=500` angefügt. Probiere es gerne bei dir lokal aus: Starte den stack, kille das backend, und gehe dann auf https://local-dev.dataland.com/api/xasd. Dann solltest du die json-formatierte Fehlermeldung sehen.